uli/README.md

```diff
@@ -3,3 +3,15 @@
 * Generate one time login url and:
   * on Mac copy it to clipboard with `pbcopy`
   * on Linux copy it to both the selection buffer and clipboard with `xclip`
+
+## Usage
+
+```
+fin uli -s @drushsitealias [ drush uli options ]
```

Comment:
@frederickjh are `-s` and `@drushsitealias` optional?

Reply:
@achekulaev yes, they are both optional. So, then they too should be in square brackets.

```suggestion
fin uli [ -s ] [ @drushsitealias ] [ drush uli options ]
```